src/Serilog.Formatting.Compact/Serilog.Formatting.Compact.csproj

```diff
@@ -1,30 +1,47 @@
-<Project Sdk="Microsoft.NET.Sdk">
+﻿<Project Sdk="Microsoft.NET.Sdk">
 
-  <PropertyGroup>
-    <Description>A simple, compact JSON-based event format for Serilog.</Description>
-    <VersionPrefix>1.1.0</VersionPrefix>
-    <Authors>Serilog Contributors</Authors>
-    <TargetFrameworks>net452;netstandard1.1;netstandard2.0</TargetFrameworks>
-    <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
-    <GenerateDocumentationFile>true</GenerateDocumentationFile>
-    <AssemblyName>Serilog.Formatting.Compact</AssemblyName>
-    <AssemblyOriginatorKeyFile>../../assets/Serilog.snk</AssemblyOriginatorKeyFile>
-    <SignAssembly>true</SignAssembly>
-    <PublicSign Condition=" '$(OS)' != 'Windows_NT' ">true</PublicSign>
-    <PackageId>Serilog.Formatting.Compact</PackageId>
-    <PackageTags>serilog;json</PackageTags>
-    <PackageIconUrl>http://serilog.net/images/serilog-extension-nuget.png</PackageIconUrl>
-    <PackageProjectUrl>https://github.com/serilog/serilog-formatting-compact</PackageProjectUrl>
-    <PackageLicenseExpression>Apache-2.0</PackageLicenseExpression>
-  </PropertyGroup>
+    <PropertyGroup>
+        <Description>A simple, compact JSON-based event format for Serilog.</Description>
+        <VersionPrefix>3.0.0</VersionPrefix>
+        <Authors>Serilog Contributors</Authors>
+        <TargetFrameworks>net6.0;netstandard2.0</TargetFrameworks>
+        <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
+        <GenerateDocumentationFile>true</GenerateDocumentationFile>
+        <AssemblyName>Serilog.Formatting.Compact</AssemblyName>
+        <AssemblyOriginatorKeyFile>../../assets/Serilog.snk</AssemblyOriginatorKeyFile>
+        <SignAssembly>true</SignAssembly>
+        <PublicSign Condition=" '$(OS)' != 'Windows_NT' ">true</PublicSign>
+        <PackageIcon>serilog-extension-nuget.png</PackageIcon>
+        <PackageId>Serilog.Formatting.Compact</PackageId>
```

Comment:
`AssemblyName`, `PackageId`, and `Title` are all correctly defaulted by the build system and don't need to be explicitly set, here.

Reply:
is it same for Serilog.Formatting.Compact.Tests proj? I removed it from Serilog.Formatting.Compact, but not yet from the *.Test